babybear/src/main/java/frc/robot/control/AutonomousChooser.java

```diff
@@ -0,0 +1,323 @@
+        command.addCommands(
+                getScoreAndDriveRoutine(trajectories.getNode2Position(), trajectories.getNode2Trajectory()));
+        command.addCommands(getBalanceRoutine(balanceChooser, trajectories.getLeftToOntoRampTrajectory()));
+        return command;
+    }
+
+    private Command getNode8Routine() {
+        SequentialCommandGroup command = new SequentialCommandGroup();
+        command.addCommands(
+                getScoreAndDriveRoutine(trajectories.getNode8Position(), trajectories.getNode8Trajectory()));
+        command.addCommands(getBalanceRoutine(balanceChooser, trajectories.getRightToOntoRampTrajectory()));
+        return command;
+    }
+
+    private Command getRightRoutine() {
+        SequentialCommandGroup command = new SequentialCommandGroup();
+        command.addCommands(
+                getScoreAndDriveRoutine(trajectories.getNode9Position(), trajectories.getRightTrajectory()));
+        command.addCommands(getBalanceRoutine(balanceChooser, trajectories.getRightToOntoRampTrajectory()));
+        return command;
+    }
+
+    /**
+     * Builds a command list for use in auto routines
+     * 
+     * @param NodePosition starting position of robot corrosponding to the node.
+     *                     Nodes are numbered from left to right 1- 9 from the
+     *                     drivers perspective
+     * @param Trajectory   trajectory to follow out of the community
+     * @return command
+     */
+    private Command getScoreAndDriveRoutine(Pose2d NodePosition, Trajectory Trajectory) {
+        SequentialCommandGroup command = new SequentialCommandGroup();
+        command.addCommands(getScoreRoutine(NodePosition, trajectories.getFirstSegmentConfig()));
+        command.addCommands(new DriveTrajectoryCommand(subsystems.getDriveTrainSubsystem(), Trajectory));
+        return command;
+    }
+
+    private void setRobotPose(SequentialCommandGroup command, Pose2d pose) {
+        command.addCommands(
+                // set yaw to the starting rotation here so that field orientation ends up
+                // correct after auto
+                new InstantCommand(() -> subsystems.getDriveTrainSubsystem().setYaw(pose.getRotation().getDegrees()),
+                        subsystems.getDriveTrainSubsystem()));
+        command.addCommands(
+                new InstantCommand(() -> subsystems.getDriveTrainSubsystem().setRobotPosition(pose),
+                        subsystems.getDriveTrainSubsystem()));
+        command.addCommands(new InstantCommand(
+                () -> System.out.println("Setting Robot Position to : " + pose)));
+    }
+
+    private enum AutonomousPath {
+        LEFT_PATH,
+        RIGHT_PATH,
+        MIDDLE_PATH,
+        DIRECT_PATH,
+        TEST_NODE5_SCORE_ROUTINE,
+        NODE2_ROUTINE,
+        NODE8_ROUTINE
+    }
+
+    private enum AutonomousBalance {
+        DO_BALANCE,
+        DO_NOT_BALANCE
+    }
+
+    private enum ScoringPosition {
+        SCORE_HIGH,
```

Comment:
AFAIK - the new robot will only score low ... so middle an especially high should be removed ... unless there was a change in planned robot capabilities.

Reply:
Robot scores low, mid, and high